### PR TITLE
Add chat tutorial source code

### PR DESCRIPTION
**Pull request by olguzzar**
Source code to accompany the [chat tutorial](https://github.com/microsoft/vscode-docs/pull/7633)

### CODE REVIEW

**Review by isidorn — DISMISSED** [review]
Looks good to me. Did you test it out that it compiles and runs? Also assigning Matt as a reviewer

**Comment by olguzzar** [comment]
> Looks good to me. Did you test it out that it compiles and runs? Also assigning Matt as a reviewer

@isidorn Yes, tested and it runs correctly!

**Review by mjbvz — COMMENTED** [review]
Please make sure to list the new sample in https://github.com/microsoft/vscode-extension-samples/blob/5651637c527e07173bd066f5fb7e171ef4616cab/.scripts/samples.js#L289 too

**Review by mjbvz — APPROVED** [review]

File: chat-tutorial/tutor.jpeg
Comment (IEvangelist) [inline chat-tutorial/tutor.jpeg:1]: Why is this icon a bit blurry towards the bottom, near the trackpad on the laptop?

File: chat-tutorial/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Chat Tutorial
+
+Visual Studio Code's Copilot Chat architecture enables extension authors to integrate with the GitHub Copilot Chat experience. A chat extension is a VS Code extension that uses the Chat extension API by contributing a Chat participant. Chat participants are domain experts that can answer user queries within a specific domain.
+
+This GitHub Copilot Extension sample shows:
+
+- How to contribute a chat participant to the GitHub Copilot Chat view.
+
+![demo](./demo.png)
```
Comment (IEvangelist) [inline chat-tutorial/README.md:9]: This image file doesn't exist. Also, if you're going to provide a demo image, please also provide a meaningful alt-text to help with accessibility. Thank you. 🤓

Comment (olguzzar) [reply chat-tutorial/README.md]: good catch, forgot to move the image over. updated alt text too!

**Review by mjbvz — APPROVED** [review]

**Pull request merged**